src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/mgr-modules/mgr-module-form/mgr-module-form.component.html

```diff
@@ -0,0 +1,123 @@
+          </label>
+
+          <!-- Field control -->
+          <!-- bool -->
+          <div class="col-sm-7"
+               *ngIf="moduleOption.value.type === 'bool'">
+            <div class="checkbox checkbox-primary">
+              <input id="{{ moduleOption.value.name }}"
+                     type="checkbox"
+                     formControlName="{{ moduleOption.value.name }}">
+              <label for="{{ moduleOption.value.name }}"></label>
+            </div>
+          </div>
+
+          <!-- addr|str|uuid -->
+          <div class="col-sm-7"
+               *ngIf="['addr', 'str', 'uuid'].includes(moduleOption.value.type)">
+            <input id="{{ moduleOption.value.name }}"
+                   class="form-control"
+                   type="text"
+                   formControlName="{{ moduleOption.value.name }}"
+                   *ngIf="moduleOption.value.enum_allowed.length === 0">
+            <select id="{{ moduleOption.value.name }}"
+                    class="form-control"
+                    formControlName="{{ moduleOption.value.name }}"
+                    *ngIf="moduleOption.value.enum_allowed.length > 0">
+              <option *ngFor="let value of moduleOption.value.enum_allowed"
+                      [ngValue]="value">
+                {{ value }}
+              </option>
+            </select>
+            <span class="help-block"
+                  *ngIf="mgrModuleForm.showError(moduleOption.value.name, frm, 'invalidUuid')"
+                  i18n>The entered value is not a valid UUID, e.g.: 67dcac9f-2c03-4d6c-b7bd-1210b3a259a8</span>
+            <span class="help-block"
+                  *ngIf="mgrModuleForm.showError(moduleOption.value.name, frm, 'pattern')"
+                  i18n>The entered value needs to be a valid IP address.</span>
+          </div>
+
+          <!-- uint|int|size|secs -->
+          <div class="col-sm-7"
+               *ngIf="['uint', 'int', 'size', 'secs'].includes(moduleOption.value.type)">
+            <input id="{{ moduleOption.value.name }}"
+                   class="form-control"
+                   type="number"
+                   formControlName="{{ moduleOption.value.name }}"
+                   min="{{ moduleOption.value.min }}"
+                   max="{{ moduleOption.value.max }}">
+            <span class="help-block"
+                  *ngIf="mgrModuleForm.showError(moduleOption.value.name, frm, 'required')"
+                  i18n>This field is required.</span>
+            <span class="help-block"
+                  *ngIf="mgrModuleForm.showError(moduleOption.value.name, frm, 'max')"
+                  i18n>The entered value is too high! It must not be lower than {{ moduleOption.value.max }}.</span>
+            <span class="help-block"
+                  *ngIf="mgrModuleForm.showError(moduleOption.value.name, frm, 'min')"
+                  i18n>The entered value is too low! It must not be greater than {{ moduleOption.value.min }}.</span>
```

Comment:
```suggestion
                  i18n>The entered value is too low! It must be greater than {{ moduleOption.value.min }}.</span>
```

Reply:
Also I entered ``24`` for the interval (within Telemetry plugin) which is accepted by the validation. So I think the validation message should rather say that the entered value must be **at least** 24 or greater.